CHANGELOG_V1.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## [v1.0.75](https://github.com/stargate/stargate/tree/v1.0.75) (2023-03-22)
+
+[Full Changelog](https://github.com/stargate/stargate/compare/v1.0.74...v1.0.75)
+
+**Closed issues:**
+
+- Use concurrency to cancel workflows on pull request updates [\#2493](https://github.com/stargate/stargate/issues/2493)
+- Update to DSE 6.8.33 [\#2490](https://github.com/stargate/stargate/issues/2490)
+- Update to DSE 6.8.33 [\#2487](https://github.com/stargate/stargate/issues/2487)
```

Comment:
Probably want to remove 2 of these?

Reply:
Oh, if these are already in V1 release notes never mind.

Reply:
Aren't these should be handled in v1 branch? else we will have conflict every time.

Reply:
Yes, correct.